SeaPublicWebsite/appsettings.json

```diff
@@ -6,5 +6,17 @@
             "Microsoft.Hosting.Lifetime": "Information"
         }
     },
-    "AllowedHosts": "*"
+    "AllowedHosts": "*",
+    
+    "GovUkNotify": {
+        "Test": "test",
+        "ApplicationReferenceNumber": {

```

Comment:
Can we make it clearer what these configuration values are for?
Maybe put these in a `"TemplateDetails": {` section? Or put `Template` at the end of each name (so `ApplicationReferenceNumberTemplate`.
Inside each section please make it clear that the other values are the names of placeholders. Maybe just start all the names with `placeholder` or similar.

Reply:
+1 For adding 'Template' at the end of the name and for the placeholder suggestion :)
If only there was a way to make the names smaller.. oh well, clarity beats bad naming!